code/datums/materials/meat.dm

```diff
@@ -80,16 +80,31 @@
 		eat_time = 3 SECONDS, \
 		tastes = list("meat" = 1))
 
-	source.AddComponent(
-		/datum/component/bloody_spreader,\
-		blood_left = (protein_count + fat_count) * 0.3 * multiplier,\
-		blood_dna = blood_dna,\
-	)
+	if (blood_dna)
+		source.AddComponent(
+			/datum/component/bloody_spreader,\
+			blood_left = (protein_count + fat_count) * 0.3 * multiplier,\
+			blood_dna = blood_dna,\
+		)
+	else
+		source.AddComponent(
+			/datum/component/bloody_spreader,\
+			blood_left = (protein_count + fat_count) * 0.3 * multiplier,\
+		)
```

Comment:
Are these not identical? If `blood_dna` is `null` we get component defaults anyways?